.github/workflows/integrationTests.yml

```diff
@@ -0,0 +1,6 @@
+on:
+  workflow_call:
+
+jobs:
+  windows-integration-tests:
+    uses: ./.github/workflows/integrationTestsWindows.yml
```

Comment:
You'll not I'm not reusing any of the shared GHA workflows.  This is primarily due to this repo using npm instead of yarn.